Emby.Server.Implementations/Localization/Core/en-US.json

```diff
@@ -131,5 +131,7 @@
     "TaskKeyframeExtractor": "Keyframe Extractor",
     "TaskKeyframeExtractorDescription": "Extracts keyframes from video files to create more precise HLS playlists. This task may run for a long time.",
     "TaskCleanCollectionsAndPlaylists": "Clean up collections and playlists",
-    "TaskCleanCollectionsAndPlaylistsDescription": "Removes items from collections and playlists that no longer exist."
+    "TaskCleanCollectionsAndPlaylistsDescription": "Removes items from collections and playlists that no longer exist.",
```

Comment:
I think new keys should be placed in alphabetical order to minimise conflicts.
Since the first part is in order, I suggest placing them after `"System"`.

Reply:
Hmm, wouldn't this make it look pretty out of place compared to existing keys? I mean I can do it